--- a/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp
+++ b/src/WallpaperEngine/WebBrowser/CEF/RenderHandler.cpp
@@ -24,4 +24,4 @@
 
 int RenderHandler::getHeight () const { return this->m_webdata->getHeight (); }
 
-GLuint RenderHandler::texture () const { return this->m_webdata->getWallpaperFramebuffer (); }
\ No newline at end of file
+GLuint RenderHandler::texture () const { return this->m_webdata->getWallpaperTexture (); }
\ No newline at end of file
